zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -2375,7 +2375,7 @@ public boolean authWriteRequest(Request request) {
                 try {
                     request.cnxn.sendResponse(rh, null, null);
                 } catch (IOException e) {
-                    LOG.error("IOException : {}", e);
+                    LOG.error("IOException", e);
```

Comment:
```suggestion
                    LOG.warn("IOException", e);
```

`error` with exception stacktrace panics, a client connection io error does not deserve that.

Currently, the handling of `sendResponse` is not consistent. I checked the code, and found no `IOException`. We probably should revise and figure out how to handle client connection error in a consistent way.

Reply:
Yes, you looked very carefully and indeed there won't be any `IOException`.😂